amazon-reviews-fasttext/amazon_pyx.ipynb

```diff
@@ -4,23 +4,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This example demonstrates the use of `cleanlab`'s experimental module `cleanlab.experimental.fasttext`\n",
-    "\n",
-    "Code is adapted from v1 examples (see `contrib/v1` folder)\n"
+    "# Amazon Reviews `fasttext` Tutorial"
```

Comment:
Let's try to avoid using the word "tutorial" anywhere in this Examples/ repo, otherwise things may get confusing. Also I don't think titles will typically use backticks \`\`.  

So can simply title this:  

Finding label issues in Amazon Reviews (Text Data) with FastText

Reply:
classifier_comparison.ipynb also has "Tutorial" in the name which we should remove